endpointmanager/scripts/populatenpiorgs.go

```diff
@@ -0,0 +1,423 @@
+	Healthcare_Provider_Taxonomy_Group_3                                   string
+	Healthcare_Provider_Taxonomy_Group_4                                   string
+	Healthcare_Provider_Taxonomy_Group_5                                   string
+	Healthcare_Provider_Taxonomy_Group_6                                   string
+	Healthcare_Provider_Taxonomy_Group_7                                   string
+	Healthcare_Provider_Taxonomy_Group_8                                   string
+	Healthcare_Provider_Taxonomy_Group_9                                   string
+	Healthcare_Provider_Taxonomy_Group_10                                  string
+	Healthcare_Provider_Taxonomy_Group_11                                  string
+	Healthcare_Provider_Taxonomy_Group_12                                  string
+	Healthcare_Provider_Taxonomy_Group_13                                  string
+	Healthcare_Provider_Taxonomy_Group_14                                  string
+	Healthcare_Provider_Taxonomy_Group_15                                  string
+}
+
+func ParseNPIdataLine(line []string) NPICsvLine {
+	data := NPICsvLine{
+		NPI:              line[0],
+		Entity_Type_Code: line[1],
+		Provider_Organization_Name_Legal_Business_Name:          line[4],
+		Provider_Other_Organization_Name:                        line[11],
+		Provider_First_Line_Business_Practice_Location_Address:  line[28],
+		Provider_Second_Line_Business_Practice_Location_Address: line[29],
+		Provider_Business_Practice_Location_Address_City_Name:   line[30],
+		Provider_Business_Practice_Location_Address_State_Name:  line[31],
+		Provider_Business_Practice_Location_Address_Postal_Code: line[32],
+		Healthcare_Provider_Taxonomy_Code_1:                     line[47],
+	}
+	return data
+}
+
+func BuildNPIOrgFromNPICsvLine(data NPICsvLine) *endpointmanager.NPIOrganization {
+	npi_org := &endpointmanager.NPIOrganization{
+		NPI_ID:        data.NPI,
+		Name:          data.Provider_Organization_Name_Legal_Business_Name,
+		SecondaryName: data.Provider_Other_Organization_Name,
+		Location: &endpointmanager.Location{
+			Address1: data.Provider_First_Line_Business_Practice_Location_Address,
+			Address2: data.Provider_Second_Line_Business_Practice_Location_Address,
+			City:     data.Provider_Business_Practice_Location_Address_City_Name,
+			State:    data.Provider_Business_Practice_Location_Address_State_Name,
+			ZipCode:  data.Provider_Business_Practice_Location_Address_Postal_Code},
+		Taxonomy: data.Healthcare_Provider_Taxonomy_Code_1}
+	return npi_org
+}
+
+func panicOnErr(err error) {
+	if err != nil {
+		panic(err)
+	}
+}
+
+func main() {
+	store, err := postgresql.NewStore(viper.GetString("dbhost"), viper.GetInt("dbport"), viper.GetString("dbuser"), viper.GetString("dbpass"), viper.GetString("dbname"), viper.GetString("dbsslmode"))
+	panicOnErr(err)
+	lines, err := ReadCsv("npidata_pfile_20050523-20191110.csv")
+	panicOnErr(err)
+
+	// Loop through lines & turn into object
+	for _, line := range lines {
```

Comment:
i'd move this looping stuff into a function and then create an additional test for this (particularly to ensure the entity code stuff)